packages/create-catalyst/tsup.config.ts

```diff
@@ -3,7 +3,6 @@ import { defineConfig, Options } from 'tsup';
 export default defineConfig((options: Options) => ({
   entry: ['src/index.ts'],
   format: ['esm'],
-  dts: true,
```

Comment:
Somehow, `@commander-js/extra-typings` causes `tsup` to no longer properly handle shebangs when generating declaration files: https://github.com/egoist/tsup/issues?q=%22%23%21%2Fusr%2Fbin%2Fenv+node%22+

```sh
pnpm build

> @bigcommerce/create-catalyst@0.1.0 build catalyst/packages/create-catalyst
> tsup

CLI Building entry: src/index.ts
CLI Using tsconfig: tsconfig.json
CLI tsup v6.7.0
CLI Using tsup config: catalyst/packages/create-catalyst/tsup.config.ts
CLI Target: es2022
CLI Cleaning output folder
ESM Build start
ESM dist/index.js 25.98 KB
ESM ⚡️ Build success in 12ms
DTS Build start
Error: Syntax not yet supported

  1 | import * as __utils_pm_js from "./utils/pm.js";
> 2 | #!/usr/bin/env node
    |  ^^^^^^^^^^^^^
  3 | import { Command } from '@commander-js/extra-typings';
  4 | type Options<T> = T extends Command<infer Args, infer Opts> ? [...Args, Opts, T] : never;
  5 | declare const createCommand: Command<[], {
```

Reply:
We don't need types exported in a CLI anyways.